sorc/grid_tools.fd/regional_esg_grid.fd/pmat4.f90

```diff
@@ -155,7 +167,11 @@ function normalized_d(a)result(b)!                                [normalized]
 s=absv_d(a); if(s==u0)then; b=u0;else;b=a/s;endif
 end function normalized_d
 
-!> calculation for orthogonalized_s function
+!> Doing calculation for orthogonalized_s function.
+!!
+!! @param[in] u real type input value
+!! @param[in] a real type input value
+!! @param[out] b result
```

Comment:
For the results, use the @return tag not the @param tag.

So:
@return b result

Reply:
Replaced with ??? because no original document support.